modules/manage/partials/recovery-mode.adoc

```diff
@@ -0,0 +1,382 @@
+endif::[]
++
+You should see a list of brokers that are in recovery mode. For example:
++
+[,bash,lines=8,.no-copy]
+----
+CLUSTER HEALTH OVERVIEW
+=======================
+Healthy:                          true
+Unhealthy reasons:                []
+Controller ID:                    0
+All nodes:                        [0 1 2]
+Nodes down:                       []
+Nodes in recovery mode:           [0 1 2]
+Leaderless partitions (0):        []
+Under-replicated partitions (0):  []
+----
+
+In recovery mode, all private Redpanda topics such as `__consumer_offsets` are accessible. Data in user-created topics is not available, but you can still manage the metadata for these topics.
+
+== Exit recovery mode
+
+ifdef::env-kubernetes[]
+Exit recovery mode by disabling it on all brokers:
+
+[tabs]
+======
+Helm + Operator::
++
+--
+.`redpanda-cluster.yaml`
+[,yaml,lines=10]
+----
+apiVersion: cluster.redpanda.com/v1alpha1
+kind: Redpanda
+metadata:
+  name: redpanda
+spec:
+  chartRef: {}
+  clusterSpec:
+    config:
+      node:
+        recovery_mode_enabled: false
+----
+
+```bash
+kubectl apply -f redpanda-cluster.yaml --namespace <namespace>
+```
+
+--
+Helm::
++
+--
+[tabs]
+====
+--values::
++
+.`recovery-mode.yaml`
+[,yaml]
+----
+config:
+  node:
+    recovery_mode_enabled: false
+----
++
+```bash
+helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --create-namespace \
+--values recovery-mode.yaml --reuse-values
+```
+
+--set::
++
+[,bash]
+----
+helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --create-namespace \
+  --set config.node.recovery_mode_enabled=false
+----
+
+====
+--
+======
+endif::[]
+
+ifndef::env-kubernetes[]
+Exit recovery mode by running one of the following commands:
+
+* To exit into developer mode:
++
+[,bash]
+----
+rpk redpanda mode developer
+----
+
+* To exit into production mode:
++
+[,bash]
+----
+rpk redpanda mode production
+----
+endif::[]
+
+== Disable partitions
+
+Problems that prevent normal cluster startup may be isolated to certain partitions or topics. You can use rpk or the xref:api:ROOT:admin-api.adoc#tag/Partitions[Admin API] to disable these partitions at the topic level, or individual partition level. A disabled partition or topic returns a `Replica Not Available` error code for Kafka API requests.
+
+To disable a partition, you need a healthy controller in the cluster, so you must start the cluster in recovery mode if a problematic partition is affecting cluster startup. If you disable a partition while in recovery mode, starting Redpanda again in non-recovery mode leaves the partition in a deactivated state. You must explicitly re-enable the partition.
+
+You can also disable a partition _outside_ of recovery mode, if the issue is localized to the partition and does not interfere with cluster startup.
+
+The following examples show you how to use the Admin API to enable or disable partitions. The examples assume that the Admin API port is `9644`.
```

Comment:
```suggestion
The following examples show you how to use the Admin API to enable or disable partitions. The examples are based on the assumption that the Admin API port is `9644`.
```

Reply:
Please implement this one^^ as example are not sentient and cannot make assumptions. ;-)